Description: Correct path to lspci
 Scripts have hardcoded path to /sbin/lspci. It is /usr/bin/lcpsi on Debian.
Author: Brian T. Smith <bsmith@systemfabricworks.com>
Forwarded: no
Last-Update: <2017-11-20>
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/CommonInstall/util_driver.pl
+++ b/CommonInstall/util_driver.pl
@@ -337,8 +337,8 @@
 # provide a verbose message during install
 sub check_pcie_link
 {
-	my $setpci = '/sbin/setpci';
-	my $lspci = '/sbin/lspci';
+	my $setpci = '/usr/bin/setpci';
+	my $lspci = '/usr/bin/lspci';
     if (open (PCI, "$lspci -d 15b3: -n|")) {
         while(<PCI>) {
             my $devinfo = $_;
--- a/CommonInstall/util_init.pl
+++ b/CommonInstall/util_init.pl
@@ -95,8 +95,8 @@
 
 
 # Command paths
-my $LSPCI = "/sbin/lspci";
-my $RPM="/bin/rpm";
+my $LSPCI = "/usr/bin/lspci";
+my $RPM="/usr/bin/rpm";
 
 # a few key commands to verify exist
 my @verify_cmds = ( "uname", "mv", "cp", "rm", "ln", "cmp", "yes", "echo", "sed", "chmod", "chown", "chgrp", "mkdir", "rmdir", "grep", "diff", "awk", "find", "xargs", "sort", $RPM, "chroot", $LSPCI );
